docs/concepts/alerts.md

```diff
@@ -0,0 +1,94 @@
+# Alerts and events
+
+Events are alerts are the vehicle for informing you about possible problem situations.
+
+- Event &mdash; something that happens in the system. An event may be a raw event (for example, drift-detection job detects drift in a given model) or a calculated event based on other events (CPU for a job is over 70% for one hour)
+- Alert &mdash; a situation where a combination of events happening needs to be notified to the user
+   - Alert config - the configuration defining the situation where an alert needs to be issued
+   - Active alert - an alert config that was triggered and is actively issuing notifications
+- Notifications &mdash; A means of notifying a user or the system of an alert, such as email, but also by invoking another event. See {ref}`notifications`.
+
+## Alerts
+
+### General configuration
+There are two variables that control the alerts behavior: (AlertConfig)
```

Comment:
it's 3 variables :). Also, not sure how relevant AlertConfig is here as we are talking about general alerts behavior